dist/vokal-ng-api.js

```diff
@@ -187,7 +187,7 @@ angular.module( "vokal.API", [ "vokal.Humps" ] )
         };
         apiConstruct.prototype.setKey = function ( key )
         {
-            this.globalHeaders.AUTHORIZATION = String( key );
+            this.globalHeaders.AUTHORIZATION = key ? String( key ) : "";
```

Comment:
Maybe explicitly allow number values here, or only check for undefined/null?  I'd like to see `0` cast as a string instead of stored as "".